### ISSUE TITLE
Login Session Length Increase

### ISSUE THREAD

**Issue by jstart** [ExternalTesters/ExternalTesters-public#7]
Basically everytime I come to external testers I have to reauthorize.  Any way to make the time out longer?

**Comment by HendX** [comment]
Agreed this would be good. I'll need to research this. I believe this explains a possible way to do it:

https://developer.github.com/guides/basics-of-authentication/#implementing-persistent-authentication

**Comment by jstart** [comment]
@HendX do you accept PRs for the actual web app? Would be happy to look into this.

**Comment by HendX** [comment]
Thanks - I appreciate it. I don't have the source on GitHub though
